R/FormulaSums.R

```diff
@@ -266,7 +266,8 @@ FormulaSums <- function(data, formula, makeNames = TRUE, crossTable = FALSE, tot
     if (makeModelMatrix) {
       if (viaSparseMatrix) {
         if (is.factor(rg1)) {
-          n_j <- max(as.integer(levels(rg1)))
+          # n_j <- max(as.integer(levels(rg1)))

```

Comment:
Same here, the comment is referring to old code.

Reply:
Tricy topic again. I keep the comment with extra comments :)